docs/Readme.md

```diff
@@ -5,9 +5,11 @@
 ## Getting Started 
 * [Getting Started with SynthDet](GettingStartedSynthDet.md)
 * [Running SynthDet in Unity Simulation](RunningSynthDetCloud.md)
+* [Dataset Insights](https://github.com/Unity-Technologies/dataset-insights)
+* [Getting Started with SynthDet Viewer AR App](https://github.com/Unity-Technologies/perception-synthdet-demo-app)
```

Comment:
This is good, but also I think we should make sure that instructions for running the AR app are added to the end of the new instructions for doing model training. I guess that is still waiting on @adason ?

Reply:
I will work with @adason on it.